version.props

```diff
@@ -1,6 +1,6 @@
 <!-- This file may be overwritten by automation. Only values allowed here are VersionPrefix and VersionSuffix.  -->
 <Project>
     <PropertyGroup>
-        <VersionPrefix>2.0.0</VersionPrefix>
+        <VersionPrefix>2.0.1</VersionPrefix>
```

Comment:
@natemcmaster What else do I need to do?

Reply:
Is this for the 2.0.6 patch (Feb)?

Reply:
@ajcvickers

Reply:
The changes in this repo are all you need. Depending on which patch release this is targeting, you also need to update aspnet/Universe and upstream repos. For example, you'll need to update aspnet/EFCore to pull this new package version.